app/views/content/steps-to-become-a-teacher/_step_4_find_a_teacher_training_course.html.erb

```diff
@@ -2,6 +2,8 @@
 
 <p>You do not need a PGCE to teach, but you do need QTS to teach in many primary and secondary schools.</p>
 
+<p>Find out about <a href="/train-to-be-a-teacher/how-to-choose-your-teacher-training-course">how to choose your teacher training course</a>.</p>
```

Comment:
@MaxineCl @caitlin-reidy This feels a little oddly worded - worth rephrasing?

Reply:
@gemmadallmandfe agree it sounds not quite right. And I'd start with what you need (a QTS) rather than what you don't.